src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -39,8 +39,9 @@
 			<ul>
 				<!-- counter -->
 				<li v-if="Number.isInteger(item.utils.counter) && item.utils.counter > 0"
-					class="app-navigation-entry-utils-counter">
-					{{ item.utils.counter }}
+					class="app-navigation-entry-utils-counter"
+				:class="{highlighted: item.utils.counter_highlighted}">
```

Comment:
identation :)

Reply:
Only people who don't use Arch make these kind of mistakes